pkgs/top-level/all-packages.nix

```diff
@@ -18962,6 +18962,8 @@ with pkgs;
 
   snpeff = callPackage ../applications/science/biology/snpeff/default.nix { };
 
+  star = callPackage ../applications/science/biology/star/default.nix { };
```

Comment:
`default.nix` is default path when calling a directory—do not specify it explicitly.

```
star = callPackage ../application/science/biology/star { };
```

Reply:
Done. Some packages in all-packages do explicitly specify this. I will try to clean them up, at least for the `science` section sometime.